tests/ui/traits/non_lifetime_binders/binder-defaults-118697.stderr

```diff
@@ -1,20 +1,20 @@
 error[E0412]: cannot find type `A` in this scope
-  --> $DIR/issue-118697.rs:4:22
+  --> $DIR/binder-defaults-118697.rs:4:22
    |
 LL | type T = dyn for<V = A(&())> Fn(());
    |                      ^ not found in this scope
 
-error: default parameter is not allowed in this binder
-  --> $DIR/issue-118697.rs:4:22
+error: higher-ranked generic parameters may not have defaults
```

Comment:
I've updated the diagnostic message since I didn't find it clear and since we're now generally rejecting defaults for higher-ranked generic parameters.